Sources/OHHTTPStubsSwift/OHHTTPStubsSwift.swift

```diff
@@ -174,7 +174,82 @@ public extension HTTPStubsResponse {
   }
 #endif
 
+/**
+ * Helper to call the stubbing function with condition and string
+ *
+ * - Parameter condition: the matcher block that determine if the request will be stubbed
+ * - Parameter string: the string to be used as stub response if the request is stubbed
+ *
+ * - Returns: The opaque `HTTPStubsDescriptor` that uniquely identifies the stub
+ *            and can be later used to remove it with `removeStub:`
+ *
+ * - SeeAlso: `OHTTPStubsResponse.init(string:, encoding:, statusCode:, headers:)`
+ */
+@discardableResult
+public func stub(
+  condition: @escaping HTTPStubsTestBlock,
+  string: String,
+  encoding: String.Encoding = .utf8,
+  statusCode: Int32 = 200,
+  headers: [AnyHashable: Any]? = nil
+) -> HTTPStubsDescriptor {
+  stub(condition: condition) { _ in
+    .init(string: string, encoding: encoding, statusCode: statusCode, headers: headers)
+  }
+}
+
+/**
+ * Helper to call the stubbing function with condition and JSON object
+ *
+ * - Parameter condition: the matcher block that determine if the request will be stubbed
+ * - Parameter jsonObject: the JSON object to be used as stub response if the request is stubbed
+ *
+ * - Returns: The opaque `HTTPStubsDescriptor` that uniquely identifies the stub
+ *            and can be later used to remove it with `removeStub:`
+ *
+ * - SeeAlso: `HTTPStubsResponse.init(jsonObject:, serializationOptions:, statusCode:, headers:)`
+ */
+@discardableResult
+public func stub(
+  condition: @escaping HTTPStubsTestBlock,
+  jsonObject: Any,
+  serializationOptions: JSONSerialization.WritingOptions = [],
+  statusCode: Int32 = 200,
+  headers: [AnyHashable: Any]? = nil
+) -> HTTPStubsDescriptor {
+  stub(condition: condition) { _ in
+    .init(
+      jsonObject: jsonObject,
+      serializationOptions: serializationOptions,
+      statusCode: statusCode,
+      headers: headers
+    )
+  }
+}
 
+/**
+ * Helper to call the stubbing function with condition and encodable object
+ *
+ * - Parameter condition: the matcher block that determine if the request will be stubbed
+ * - Parameter encodable: the encodable object to be used as stub response if the request is stubbed
+ *
+ * - Returns: The opaque `HTTPStubsDescriptor` that uniquely identifies the stub
+ *            and can be later used to remove it with `removeStub:`
+ *
+ * - SeeAlso: `HTTPStubsResponse.init(encodable:, encoder:, statusCode:, headers:)`
+ */
+@discardableResult
+public func stub<T>(
+  condition: @escaping HTTPStubsTestBlock,
+  encodable: T,
+  encoder: JSONEncoder = .init(),
+  statusCode: Int32 = 200,
+  headers: [AnyHashable: Any]? = nil
+) -> HTTPStubsDescriptor where T: Encodable {
+  stub(condition: condition) { _ in
+    .init(data: try! encoder.encode(encodable), statusCode: statusCode, headers: headers)
```

Comment:
you should use the initializer that gets encodable, not the one that gets `Data`. If i'm wrong, then the comment about `!` is relevant here too.